src/pages/workspace/distanceRates/PolicyDistanceRateDetailsPage.tsx

```diff
@@ -48,8 +48,9 @@ function PolicyDistanceRateDetailsPage({policy, route}: PolicyDistanceRateDetail
     const customUnit = customUnits[Object.keys(customUnits)[0]];
     const rate = customUnit?.rates[rateID];
     const currency = rate?.currency ?? CONST.CURRENCY.USD;
-    const canDeleteRate = Object.values(customUnit?.rates ?? {}).filter((distanceRate) => distanceRate?.enabled).length > 1 || !rate?.enabled;
-    const canDisableRate = Object.values(customUnit?.rates ?? {}).filter((distanceRate) => distanceRate?.enabled).length > 1;
+    const canDisableorDeleteRate = Object.values(customUnit?.rates).some(
```

Comment:
Can you please add a comment here which explains why this logic is OK for controlling if a rate can be disabled or deleted? Something like:

```
Rates can be disabled or deleted as long as the rate is currently enabled and there is no pending delete action.
```

Note: I'm not exactly sure what the `rateID !== distanceRate?.customUnitRateID` logic is for, but please add that to the comment also.

Also, I believe the variable name needs the "or" capitalized like `canDisableOrDeleteRate`